packages/react/label/src/Label.tsx

```diff
@@ -12,7 +13,8 @@ import type * as Radix from '@radix-ui/react-primitive';
 const NAME = 'Label';
 
 type LabelContextValue = { id: string; ref: React.RefObject<HTMLSpanElement> };
-const LabelContext = React.createContext<LabelContextValue | undefined>(undefined);
+
+const [LabelProvider, useLabelContextImpl] = createContext<LabelContextValue | null>(NAME, null);
```

Comment:
maybe this should be `useLabelContext` and the public one `useLabel` because it does more than just returning context?

Reply:
I thought that too but then that's a breaking change.... how do we handle semver with breaking changes now that we're in beta 😕🤔